v5_enum.go

```diff
@@ -75,6 +75,8 @@ type OrderFilter string
 const (
 	// OrderFilterOrder :
 	OrderFilterOrder = OrderFilter("Order")
+	// StopOrderFilterOrder :
+	StopOrderFilterOrder = OrderFilter("StopOrder")
```

Comment:
nits:

To analogize from the type name.

```suggestion
	// OrderFilterStopOrder :
	OrderFilterStopOrder = OrderFilter("StopOrder")
```